vita3k/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 set(VITA3K_ORG_NAME "Vita3K")
 set(VITA3K_APP_NAME "Vita3K")
-set(VITA3K_APP_VERSION "v0.1")
+set(VITA3K_APP_VERSION "v0.0.2")
```

Comment:
🤔 We're going back a few versions?

Reply:
i just is to early for can bump on 0.2, so i have thinks using like rpcs3? for can bump  version more quickely, seriously i have set this 0.1 in 2018 hmhm lol